Formula/n98-magerun.rb

```diff
@@ -0,0 +1,17 @@
+class N98Magerun < Formula
+  desc "Swiss army knife for Magento developers, sysadmins and devops."
+  homepage "http://magerun.net/"
+  url "https://files.magerun.net/n98-magerun.phar"
```

Comment:
Just to check, is this a versioned URL or does it always point to the latest release?